example_config.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Great idea to name this something different than `config.json` so that we avoid committing personal config files!